packages/devtools/app/scripts/components/Code/Code.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react';
+import { Box } from '@chakra-ui/react';
+import { useColorScheme } from '../../hooks/useColorScheme';
+import Highlight, { defaultProps } from 'prism-react-renderer';
+import { useCodeTheme } from './finchTheme';
+
+interface CodeProps {
+  code: string;
+  language: 'json' | 'graphql';
+}
+
+export const Code: FC<CodeProps> = ({ code, language }) => {
+  const scheme = useColorScheme();
+  const theme = useCodeTheme();
+  return (
+    <Box as="pre" p={4} rounded={8} mb={3} overflow="scroll">
+      <Box
+        as="code"
+        color={scheme.highlightSecondary}
+        overflow="scroll"
+        whiteSpace="break-spaces"
+      >
+        <Highlight
+          {...defaultProps}
+          theme={theme}
+          code={code}
+          language={language}
+        >
```

Comment:
That's a cool component!

Reply:
Yeah found it because it's used in [react-live](https://github.com/FormidableLabs/react-live), and is being used in the [beta react site](https://beta.reactjs.org/).